src/main/java/ploopy/PloopyException.java

```diff
@@ -0,0 +1,14 @@
+package ploopy;
+
+/**
+ * An exception class that encapsulates the error message during
+ * the execution of the program.
+ */
+public class PloopyException extends Exception {
```

Comment:
Perhaps you could make your exception name more descriptive, as to what kind of an error it is?